tagstudio/src/qt/ts_qt.py

```diff
@@ -189,9 +186,42 @@ def __init__(self, core, args):
 
         self.SIGTERM.connect(self.handleSIGTERM)
 
-        self.settings = QSettings(
-            QSettings.IniFormat, QSettings.UserScope, "tagstudio", "TagStudio"
-        )
+        if self.args.config_file:
+            path = Path(self.args.config_file)
+            if path.is_dir():
+                path = path / "TagStudio.ini"
+                self.settings = QSettings(str(path), QSettings.IniFormat)
+                logging.info(
+                    f"[QT DRIVER] Directory provided defaulting to TagStudio.ini in directory, using {self.settings.fileName()}"
+                )
```

Comment:
I wouldnt do this. The user should provide valid path to existing config file, or it should fail loudly. Imagine the user makes a typo in the path (wrong autocomplete  choice etc.) and instead of `/path/to/TagStudio.ini`, they would type `/path/to/Tarkov/`, so it would create new empty config there. Most likely not desired behaviour.

Reply:
I think the only case that needs to be handled is how to create a .ini file from the command line argument then. If I want to use a new config location then I shouldn't need to go there and manually create the config file, I should be able to point the program at where I want it to save the config and then it creates the blank config file there.